server/flags.go

```diff
@@ -21,9 +21,7 @@ func AddFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(config.FlagLight, def.Node.Light, "run light client")
 	cmd.Flags().Duration(config.FlagBlockTime, def.Node.BlockTime.Duration, "block time (for aggregator mode)")
 	cmd.Flags().String(config.FlagTrustedHash, def.Node.TrustedHash, "initial trusted hash to start the header exchange service")
-	cmd.Flags().Bool(config.FlagLazyAggregator, def.Node.LazyAggregator, "produce blocks only when transactions are available or after lazy block time")
```

Comment:
We still need these. They were changed. To lazymode and lazyblock interval

Reply:
ah ok, thanks